x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/rare_view/detector_description.tsx

```diff
@@ -63,49 +58,82 @@ function createDetectorDescription(jobCreator: RareJobCreator, detectorType: RAR
   const populationFieldName = jobCreator.populationField?.id;
   const splitFieldName = jobCreator.splitField?.id;
 
-  const beginningSummary = i18n.translate(
-    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.beginningSummary',
+  const rareSummary = i18n.translate(
+    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.rareSummary',
     {
-      defaultMessage: 'detects rare values of {rareFieldName}',
+      defaultMessage: 'Detects rare {rareFieldName} values.',
       values: { rareFieldName },
     }
   );
 
-  const beginningSummaryFreq = i18n.translate(
-    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.beginningSummaryFreq',
+  const rareSplitSummary = i18n.translate(
+    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.rareSplitSummary',
     {
-      defaultMessage: 'detects frequently rare values of {rareFieldName}',
-      values: { rareFieldName },
+      defaultMessage: 'For each {splitFieldName} value, detects rare {rareFieldName} values.',
+      values: { splitFieldName, rareFieldName },
     }
   );
 
-  const population = i18n.translate(
-    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.population',
+  const freqRarePopulationSummary = i18n.translate(
+    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.freqRarePopulationSummary',
     {
-      defaultMessage: 'compared to the population of {populationFieldName}',
-      values: { populationFieldName },
+      defaultMessage:
+        'Detects {populationFieldName} values that frequently have rare {rareFieldName} values relative to the population.',
+      values: { populationFieldName, rareFieldName },
     }
   );
 
-  const split = i18n.translate('xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.split', {
-    defaultMessage: 'for each value of {splitFieldName}',
-    values: { splitFieldName },
-  });
+  const freqRareSplitPopulationSummary = i18n.translate(
+    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.freqRareSplitPopulationSummary',
+    {
+      defaultMessage:
+        'For each {splitFieldName}, detects {populationFieldName} values that frequently have rare {rareFieldName} values relative to the population.',
+      values: { splitFieldName, populationFieldName, rareFieldName },
+    }
+  );
+
+  const rarePopulationSummary = i18n.translate(
+    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.rarePopulationSummary',
+    {
+      defaultMessage:
+        'Detects {populationFieldName} values that have rare {rareFieldName} values relative to the population.',
+      values: { populationFieldName, rareFieldName },
+    }
+  );
+
+  const rareSplitPopulationSummary = i18n.translate(
+    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.rareSplitPopulationSummary',
+    {
+      defaultMessage:
+        'For each {splitFieldName} value, detects {populationFieldName} values that have rare {rareFieldName} values relative to the population.',
```

Comment:
As above, I prefer this without the first `value`.

![image](https://user-images.githubusercontent.com/7405507/125785987-9da8fd6a-08cc-4070-ac9b-af64b68430ed.png)

```
For each client.geo.region_iso_code, detects user.name values that have rare host.name values relative to the population.
```
reads better to me than
```
For each client.geo.region_iso_code value, detects user.name values that have rare host.name values relative to the population.
```

Reply:
Agreed! 19fdfd5678461ec4ac2bcaa18ea718a51bdec201